src/lineup/internal/export.ts

```diff
@@ -48,14 +48,14 @@ export function exportxlsx(columns: Column[], rows: IDataRow[]) {
   const converted = rows.map((row) => {
     const r: any = {};
     for (const col of columns) {
-      r[col.label] = isNumberColumn(col) ? col.getRawNumber(row) : col.getValue(row);
+      r[`${col.label}${col.description ? `\n${col.description}` : ''}`] = isNumberColumn(col) ? col.getRawNumber(row) : col.getValue(row);
```

Comment:
- [ ] Replace template string with normal concatinated string, since we only need a string for the `\n`
- [ ] Extract the generation of the key/name into a small separate function (similar to the `isDateColumn()` above)
- [ ] Use the new key function in line 29, 51, and 58

Reply:
Done !